packages/user-service/README.md

```diff
@@ -0,0 +1,20 @@
+User Microservice
+=================================================
+
+One platform's server-side User GraphQL microservice.
```

Comment:
Can you add some more information about this microservice? What it's for, what it does and so on?

Reply:
Updated the information